src/PopUpForm/PopUpForm.css

```diff
@@ -1,78 +1,129 @@
-@import url('https://fonts.googleapis.com/css?family=EB+Garamond|Pacifico');
-@import url('https://fonts.googleapis.com/css?family=Kaushan+Script');
+@import url('https://fonts.googleapis.com/css?family=Amatic+SC');
+@import url('https://fonts.googleapis.com/css?family=PT+Sans');
 
 
 .moodForm{
+  z-index: 10;
   background: #fbead2;
   position: absolute;
-  top: 30%;
-  left: 35%;
+  top: 20%;
+  left: 33%;
   width: 300px;
-  height: 380px;
+  height: 390px;
   border-radius: 5px;
   box-shadow: 0px 2px 36px 0px #f9e2c1;
 }
 
+span{
```

Comment:
It's rare you want to apply CSS rules to html elements like this. Better to give things classnames and give properties to classnames.

Sometimes, a library you use might have a span, and the rule unintentionally applies to that.